mikrotik-backup/backup.sh

```diff
@@ -2,11 +2,11 @@
 # usage:
 # ./hamwan_routers.sh | ./backup.sh
 
-DIR=$(dirname "${BASH_SOURCE[0]}")/backup
+DIR=/srv/router-backup
 LIMIT=8
-COMMON_OPTS="-o ConnectTimeout=10 -o BatchMode=yes"
-SCP_OPTS="$COMMON_OPTS"
-SSH_OPTS="$COMMON_OPTS -n"
+COMMON_OPTS="-o ConnectTimeout=10 -o BatchMode=yes -o StrictHostKeyChecking=no -i /var/www/.ssh/id_rsa -o User=monitoring"
+SCP_OPTS="$COMMON_OPTS -P 222"
+SSH_OPTS="$COMMON_OPTS -n -p 222"
```

Comment:
The existing design was that the port would be set based on patterns in the `ssh_config`. That seems a little more flexible that hardcoding it like this.

Reply:
I don't see that actually configured anywhere, and specifically not in /var/www/.ssh/config.  Was this change lost somewhere?  I have added "Port 222" to the file and will remove this change.